common/telemetry.go

```diff
@@ -18,10 +18,14 @@ import (
 	"context"
 	"errors"
 	"fmt"
+
+	"go.opentelemetry.io/otel/metric"
 )
 
 type ShutdownFn func(ctx context.Context) error
 
+var defaultLatencyDistribution = metric.WithExplicitBucketBoundaries(1, 2, 3, 4, 5, 6, 8, 10, 13, 16, 20, 25, 30, 40, 50, 65, 80, 100, 130, 160, 200, 250, 300, 400, 500, 650, 800, 1000, 2000, 5000, 10000, 20000, 50000, 100000)
```

Comment:
Can we add the comment what these number indicate, is time or percentage?

Reply:
Done.